src/marten/db/field/slug.cr

```diff
@@ -14,11 +20,27 @@ module Marten
           @null = false,
           @unique = false,
           @index = true,
-          @db_column = nil
+          @db_column = nil,
+          @slugify : Symbol? = nil,
+          @slugify_cb : (::String -> ::String) = ->(value : ::String) { generate_slug(value) }
         )
+          if @slugify
+            @null = true
+            @blank = true
+          end
```

Comment:
I don't think that we should set `null: true` if the `slugify` option is null. Doing so will make the field nullable at the database level, but this may not be wanted! A user should be able to use the `slugify` option and have a non-nullable field column!